trusted_applet/main.go

```diff
@@ -78,10 +77,16 @@ func main() {
 	//
 	// In this example the sent configuration is always updated with the
 	// received one.
-	if err := syscall.Call("RPC.Config", *cfg, cfg); err != nil {
-		log.Fatalf("TA configuration error, %v", err)
+	var cfgResp []byte
+	if err := syscall.Call("RPC.Config", cfg.Bytes(), &cfgResp); err != nil {
+		log.Printf("TA configuration error, %v", err)
 	}
 
+	if cfgResp != nil {
```

Comment:
What if it's null? Do I understand correctly that this config is not used, we simply parse it to make sure it's valid?

Reply:
If it's `nil` then we'll just use the default config values set in `cfg` above, if it's not `nil` (e.g. because the operator used the `witnessctl`/<something> to configure the device) then we unmarshal it into `cfg` overriding the defaults with the configured values.

Reply:
Thanks!